Component/Text-Utilies/text-utilities.css

```diff
@@ -0,0 +1,68 @@
+@import url(/CSS/docs.css);
+
+html {
+    font-size: 62.5%;
+  }
+ 
+body {
+    font-family: 'Mate', serif;
+}
+
+
+.main-content-display{
+    display: flex;
+    flex-direction: column;
+    gap:1rem;
+}
+.heading-1,.heading-2,hr{
+    color:var(--primary)
+}
+
+.grey-text{
+    color:var(--grey-text);
+}
+
+.center-text{
+    text-align: center;
+}
+
+.para{
+    text-align: justify;
+}
+
+.list{
+    list-style: none;
+    margin:1rem 0;
+    font-size:1.655rem;
+    width:100%;
+}
+
+.link {
+    text-decoration: none;
+    cursor: pointer;
+    color: inherit;
+  }
+
+.misc{
+    padding-left: 2rem;
+}
+.misc li{
+    list-style: disc;
+ }  
+
+ @media screen and (max-width : 640px){
+
+    html{
+        font-size:40%;
+    }
+
+}
+
+/* @media screen and (min-width : 1536px){
+
+    html{
+        font-size:150%;
+    }
+} 
+  */
```

Comment:
Suggestion: Remove unused comments.

Reply:
> Suggestion: Remove unused comments.

Hey, I wanted to make my site responsive for bigger screens as well. I haven't yet figured it out, hence I kept them. Is it fine?